src/components/EventManagement/EventAttendance/EventStatistics.tsx

```
@@ -0,0 +1,498 @@
+  }, [eventLabels, attendeeCounts, maleCounts, femaleCounts, otherCounts]);
+
+  const exportDemographicsToCSV = useCallback(() => {
+    const headers = [selectedCategory, 'Count'];
+    const data = [
+      headers,
+      ...categoryLabels.map((label, index) => [label, categoryData[index]]),
+    ];
+    exportToCSV(data, `${selectedCategory.toLowerCase()}_demographics.csv`);
+  }, [selectedCategory, categoryLabels, categoryData]);
+
+  const handleExport = (eventKey: string | null): number | void => {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)